### PR TITLE
[A11y] Fixes tag link color in search results

### PR DESCRIPTION
**Pull request by advay26**
Fixes https://github.com/NuGet/NuGetGallery/issues/8921

When viewing package search results on nuget.org, the links for the tags were indistinguishable from non-hyperlink text around it:
 
image

This PR fixes it and changes the tag hyperlink color such that users can differentiate between regular text and the links:

image

### CODE REVIEW

**Comment by joelverhagen** [comment]
Instead of overriding, can we just remove this `:not` based style?
image

File: src/Bootstrap/less/theme/base.less [hunk 1]
```diff
@@ -335,8 +335,8 @@ img.reserved-indicator-icon {
   line-height: @package-list-line-height;
   color: @gray-light;
 
-  a:not(.more-tags) {
-    color: @gray-light;
+  a {
+    color: @link-color;
```
Comment (joelverhagen) [inline src/Bootstrap/less/theme/base.less]: Is this needed? This seems like it should be the default link behavior in Bootstrap. I don't know all of the other styles applying to these elements so I may be totally wrong.

Comment (joelverhagen) [reply src/Bootstrap/less/theme/base.less]: Oops, nevermind :)

Comment (advay26) [reply src/Bootstrap/less/theme/base.less]: Nope, you're right. I realized I could've simplified it further

Comment (advay26) [reply src/Bootstrap/less/theme/base.less]: Fixed it now

**Review by joelverhagen — APPROVED** [review]

**Pull request merged**